RogerMendez29/cafe-website
# Feat/redux

Added redux for state management and remove previous state management logic.

Testing Steps:
- `npm i`
- `npm run start`
- Make sure all previous features work (adding to cart, logging in and out, removing from cart, etc...)
